OTVision/version.py

```diff
@@ -0,0 +1,10 @@
+def otdet_version() -> str:
+    return "1.0"
```

Comment:
What is the first version of the output format?
1.0 or 0.1?

Reply:
Result of todays review is: 1.0 😊